include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -0,0 +1,69 @@
+/*
+ *  Copyright (c) Intel Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef _ONEDPL_SERIAL_NUMERIC_IMPL_H
+#define _ONEDPL_SERIAL_NUMERIC_IMPL_H
+
+#if __cplusplus < 201703L
+#    define _ONEDPL_HAS_NUMERIC_SERIAL_IMPL 1
+#else
+// std::reduce is not available in libstdc++7, libstdc++8
+#    if __GLIBC__ && (_GLIBCXX_RELEASE < 9)
```

Comment:
I think it's better to replace `__GLIBC__` with `__GLIBCXX__`.
Is it available since 9.1 or since newer versions of 9? What if we call the code with libstdc++9.1?

Reply:
Hi, @capatober 
You are correct, std::reduce is added into libstdc++-9.3, we can't provide our own implementation if libstdc++-9.1/2 is used.
However, __GLIBCXX_RELEASE is for libstdc++ major version, do you know any way to check minor version for libstdc++?
Thanks very much.

Reply:
Please, look at the https://github.com/oneapi-src/oneDPL/blob/main/include/oneapi/dpl/pstl/onedpl_config.h#L173 for example. Since, the date is better way we could find if there are no feature macros